File: benchmarks/cpu/primes/primes.rb
def is_prime(n)
  return false if n < 2
  d = 2
  while d * d <= n
    return false if n % d == 0
    d += 1
  end
  true
end

count = 0
(2..200000).each do |n|
  count += 1 if is_prime(n)
end
puts count

File: benchmarks/cpu/primes/primes.js
function isPrime(n) {
    if (n < 2) return false;
    let d = 2;
    while (d * d <= n) {
        if (n % d === 0) return false;
        d += 1;
    }
    return true;
}

let count = 0;
for (let n = 2; n <= 200000; n++) {
    if (isPrime(n)) count += 1;
}
console.log(count);
